samples/CppArExample/CppArExample.cpp

```diff
@@ -80,6 +80,12 @@ void CppArExample::setSceneView(SceneQuickView* sceneView)
   emit sceneViewChanged();
 }
 
+// Set the location tracking mode
+void CppArExample::setTrackingMode(int mode)
```

Comment:
If we register the type for Qml, we should be able to take the actual enum here. Does that work?